e2e/playwright/fixtures/editorFixture.ts

```diff
@@ -31,19 +32,32 @@ export class EditorFixture {
 
   private _expectEditorToContain =
     (not = false) =>
-    (
+    async (
       code: string,
       {
         shouldNormalise = false,
         timeout = 5_000,
       }: { shouldNormalise?: boolean; timeout?: number } = {}
     ) => {
+      const wasPaneOpen = await this.isPaneOpen()
+      if (!wasPaneOpen) {
+        await this.openPane()
+      }
+      const resetPane = async () => {
+        if (!wasPaneOpen) {
+          await this.closePane()
+        }
+      }
       if (!shouldNormalise) {
         const expectStart = expect(this.codeContent)
         if (not) {
-          return expectStart.not.toContainText(code, { timeout })
+          const result = await expectStart.not.toContainText(code, { timeout })
+          await resetPane()
+          return result
```

Comment:
These changes make the editor fixture automatically and open and close if it's closed when `expectEditor` is called.